src/Sarif.Viewer.VisualStudio.Core/Options/SarifViewerOptionsControlResources.xaml

```diff
@@ -6,4 +6,6 @@
 
     <system:String x:Key="SarifViewerOptionsControl_MonitorSarifFolder">Load .sarif files from the .sarif folder automatically</system:String>
 
+    <system:String x:Key="SarifViewerOptionsControl_EnableGhas">Enable GitHub Advanced Security integration</system:String>
```

Comment:
>Enable GitHub Advanced Security integration [](http://example.com/codeflow?start=64&length=43)

Do devs using Visual Studio associate GHAS with CodeQL? Does it include anything besides CodeQL? This is not a final suggestion, just opening the discussion. Would a simple update like:
`Enable GitHub Advanced Security integration (ie. automatically load CodeQL results)` make this more meaningful?

I've provided the link to this PR to Courtney to review the user facing string.